app/controllers/course/leaderboards_controller.rb

```diff
@@ -3,6 +3,7 @@ class Course::LeaderboardsController < Course::ComponentController
   before_action :check_component
   before_action :load_settings
   before_action :add_leaderboard_breadcrumb
+  helper Course::Achievement::ControllerHelper.name.sub(/Helper$/, '')
```

Comment:
I'll do it later, after I've done the fixes to this PR. Have created an issue for this #983